Gemfile

```diff
@@ -51,6 +50,14 @@ gem 'sidekiq-statsd', '0.1.5'
 
 gem 'deprecated_columns', '0.1.0'
 
+if ENV['GDS_API_ADAPTERS_DEV']
```

Comment:
I think we should replace all of this with just the one line:
`gem 'gds-api-adapters', git: 'https://github.com/alphagov/gds-api-adapters.git', branch: 'publishing-api-v2-test-helper-improvements'`